_docs/2.0.0/usage/menas-conformance-rules.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: docs
+title: Usage - Menas Conformance Rules
+version: '2.0.0'
+categories:
+    - '2.0.0'
+    - usage
+---
+
+## Table Of Contents
+<!-- toc -->
+- [Table Of Contents](#table-of-contents)
+- [Intro](#intro)
+- [Casting Conformance Rule](#casting-conformance-rule)
+- [Concatenation Conformance Rule](#concatenation-conformance-rule)
```

Comment:
I think that the Coalesce Conformance rule should be added.

Reply:
Done

Reply:
I didn't want to bother Sasa with it. Planned to add it after merge. Thanks Sasa

Reply:
I cheated. I used what you wrote in Release Notes 🚀